core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1383,7 +1427,7 @@ export class LoggedOutUser extends BaseUser {
       // This reload is required to ensure the language dropdown is visible in mobile view,
       // if the earlier movements of the page have hidden it and since the inbuilt
       // scrollIntoView function call of the clickOn function didn't work as expected.
-      await this.page.reload();
+      await this.page.reload({waitUntil: 'domcontentloaded'});
```

Comment:
This means there's something wrong with the page such that it's taking really long to get ready because of which it is timing out?
If that's the case please file an issue for the same. 

Also, I am not sure, why at all reload is needed here, scrollIntoView() is unlikely to fail.

Reply:
What error it shows up? 
Something like this - "JS handle taking too long to be clickable"?

Reply:
Ah, fine. 

Use this -- 

```      await this.page.evaluate(() => {
        window.scrollTo(0, 0);
      });```
      
   and make sure to update the comments above.

Reply:
No worries, I have tested it in my local, it works.

This is so because, we won't expect a user to reload instead of scrolling back on top to get to see the language dropdown.

Overall, the changes now looks good to me. I edited tests passes the CI.

Thanks @AkashPaloju .

Reply:
Also, sorry, but you perhaps missed to address one of the concerns above: 

> This means there's something wrong with the page such that it's taking really long to get ready because of which > it is timing out?
> If that's the case please file an issue for the same.

Reply:
Please make to sure file an issue and reference that here.